Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Search.hs

```diff
@@ -114,6 +116,7 @@ handler merchantId sReq = do
     throwError RideNotServiceable
   result <- getDistanceAndDuration merchantId fromLocationLatLong toLocationLatLong sReq.routeInfo
   CD.cacheDistance sReq.transactionId (result.distance, result.duration)
+  Redis.setExp CD.deviceKey sReq.device 120
```

Comment:
is this key common for all search request if not then why are you not adding search request id in key?